test/test_torch.py

```diff
@@ -16454,38 +16454,38 @@ def test_svd_tall_all_col_maj(self, device, dtype):
-    'exp': ufunc_meta(is_floating=True, complex_on=('cpu'), complex_out_cpu=True),
-    'expm1': ufunc_meta(is_floating=True, complex_on=()),
-    'log': ufunc_meta(is_floating=True, complex_on=('cpu'), complex_out_cpu=True),
-    'log10': ufunc_meta(is_floating=True, complex_on=('cpu'), complex_out_cpu=True),
-    'log1p': ufunc_meta(is_floating=True, complex_on=()),
-    'log2': ufunc_meta(is_floating=True, complex_on=('cpu'), complex_out_cpu=True),
-    'sqrt': ufunc_meta(is_floating=True, complex_on=('cpu'), complex_out_cpu=True),
-    'rsqrt': ufunc_meta(is_floating=True, complex_on=('cpu'), complex_out_cpu=True),
-    'trunc': ufunc_meta(is_floating=True, complex_on=('cpu'), complex_out_cpu=True),
-    'erf': ufunc_meta(is_floating=True, complex_on=()),
-    'erfc': ufunc_meta(is_floating=True, complex_on=()),
-    'erfinv': ufunc_meta(is_floating=True, complex_on=()),
-    'lgamma': ufunc_meta(is_floating=True, complex_on=()),
-    'digamma': ufunc_meta(is_floating=True, complex_on=()),
-    'sigmoid': ufunc_meta(is_floating=True, complex_on=('cpu'), complex_out_cpu=True)
+    'sin': ufunc_meta(is_floating=True, complex_on=('cpu')),
+    'cos': ufunc_meta(is_floating=True, complex_on=('cpu'), complex_out=('cpu')),
+    'tan': ufunc_meta(is_floating=True, complex_on=('cpu'), complex_out=('cpu')),
+    'asin': ufunc_meta(is_floating=True, complex_on=('cpu')),
+    'acos': ufunc_meta(is_floating=True, complex_on=('cpu')),
+    'atan': ufunc_meta(is_floating=True, complex_on=('cpu'), complex_out=('cpu')),
+    'sinh': ufunc_meta(is_floating=True, complex_on=('cpu')),
+    'cosh': ufunc_meta(is_floating=True, complex_on=('cpu'), complex_out=('cpu')),
+    'tanh': ufunc_meta(is_floating=True, complex_on=('cpu'), complex_out=('cpu')),
+    'ceil': ufunc_meta(is_floating=True, complex_on=('cpu')),
+    'floor': ufunc_meta(is_floating=True, complex_on=('cpu')),
+    'exp': ufunc_meta(is_floating=True, complex_on=('cpu'), complex_out=('cpu')),
+    'expm1': ufunc_meta(is_floating=True, complex_on=(), complex_out=()),
+    'log': ufunc_meta(is_floating=True, complex_on=('cpu')),
+    'log10': ufunc_meta(is_floating=True, complex_on=('cpu')),
+    'log1p': ufunc_meta(is_floating=True, complex_on=(), complex_out=()),
+    'log2': ufunc_meta(is_floating=True, complex_on=('cpu')),
+    'sqrt': ufunc_meta(is_floating=True, complex_on=('cpu')),
+    'rsqrt': ufunc_meta(is_floating=True, complex_on=('cpu')),
+    'trunc': ufunc_meta(is_floating=True, complex_on=('cpu')),
+    'erf': ufunc_meta(is_floating=True, complex_on=(), complex_out=()),
+    'erfc': ufunc_meta(is_floating=True, complex_on=(), complex_out=()),
+    'erfinv': ufunc_meta(is_floating=True, complex_on=(), complex_out=()),
+    'lgamma': ufunc_meta(is_floating=True, complex_on=(), complex_out=()),
+    'digamma': ufunc_meta(is_floating=True, complex_on=(), complex_out=()),
+    'sigmoid': ufunc_meta(is_floating=True, complex_on=('cpu'), complex_out=('cpu'))
```

Comment:
These changes in the `complex_out` arguments are after the PR #34093 (which adds type promotion logic for complex numbers) is merged. 

Unary Floating Ufuncs which do not support complex tensors as input/out argument on:

* CPU devices: `expm1, log1p, erf, erfc, erfinv, lgamma, digamma`.
* CUDA devices: `cos, tan, tanh, cosh, atan, exp, expm1, log1p, erf, erfc, erfinv, lgamma, digamma, sigmoid`.